src/translations/en.json

```diff
@@ -1784,8 +1784,6 @@
             "listen_to_events": "Listen to events",
             "listening_to": "Listening to",
             "subscribe_to": "Event to subscribe to",
-            "start_listening": "Start listening",
-            "stop_listening": "Stop listening",
```

Comment:
The two are used in `event-subscribe-card.ts`

Reply:
Sorry thought you linked to them as well and that I did not use them. Will fix it immediately.